api/serializers/songbook.py

```diff
@@ -1,3 +1,6 @@
+from datetime import datetime
```

Comment:
We didn't end up using these three lines, can you remove?

Reply:
Done.